connectivity/tests/perfpod.go

```diff
@@ -42,17 +42,21 @@ func (s *netPerfPodtoPod) Name() string {
 func (s *netPerfPodtoPod) Run(ctx context.Context, t *check.Test) {
 	samples := t.Context().Params().PerfSamples
 	duration := t.Context().Params().PerfDuration
+	crr := t.Context().Params().PerfCRR
 	for _, c := range t.Context().PerfClientPods() {
 		c := c
 		for _, server := range t.Context().PerfServerPod() {
-			action := t.NewAction(s, "iperf-tcp", &c, server)
+			action := t.NewAction(s, "netperf", &c, server)
```

Comment:
nit: This changes the benchmarking tool, which is not mentioned in the commit message. Is this change intentional?